tests/ClientTest.php

```diff
@@ -100,8 +100,19 @@ private function getPostEventData(): Event
         return new Event(
             'job_failed',
             new FailedJobEventData(
```

Comment:
Tady by bylo idelani mit neco jako `FailedJobEventData->getEventName()` (resp. asi s nejakym interface ala `EventDataInterface`). Tim padem nebude potrebovat do eventu davat oddelene nazev eventu a data a nestane se ti, ze by si pomotal event se spatnymi daty.

Reply:
jo, to je dobrý, aspoň ubyl jeden parametr